test/Microsoft.TestPlatform.AcceptanceTests/CodeCoverageAcceptanceTestBase.cs

```diff
@@ -59,7 +59,7 @@ protected XmlNode GetNode(XmlNode node, string type, string name)
         return node.SelectSingleNode($"//{type}[@name='{name}']") ?? node.SelectSingleNode($"//{type}[@name='{name.ToLower()}']");
     }
 
-    protected XmlDocument GetXmlCoverage(string coverageResult, TempDirectory tempDirectory)
+    protected XmlDocument GetXmlCoverage(string coverageResult, TempDirectory TempDirectory)
```

Comment:
I don't think you want this rename. Also, do we want to give the possibility to pass a temp dir or shall we use the base property directly within the method?

Reply:
I reverted the rename. How temp directory is passed is up the to test author, I am not trying to change everything in this PR.